### DISCUSSION TITLE
Bloatynosy 1.5.0

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#125] [Announcements]
Functional tests and adjustments are being conducted so that some components may be incorporated into the new [MSBob app](), and/or to ensure compatibility with Bloatynosy as well. More information about the project can be found by following the link.
https://github.com/builtbybel/MSBob

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Bloatynosy/releases/tag/1.5.0'>Bloatynosy 1.5.0</a>.</em>

**Comment by deus-e-x** [comment]
> Just a quick question, do we still need the project packaged as a setup (MSI)?

The last time you made an MSI package available was more than a year ago. Did anyone complain? I for one don't need the installer at all.

**Comment by dan-i-welcome** [comment]
Thanks for the app, but the UI is horrible - there are no checkboxes to choose which settings to apply and which not, and the "trash bin" concept is unclear - are these the settings that will be applied or the ones that will not be applied?

a simple list with checkboxes would be a lot better.

thanks
